sgt-back-end/index.js

```diff
@@ -0,0 +1,133 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const jsonMiddleWare = express.json();
+
+app.use(jsonMiddleWare);
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+      from grades
+  `;
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.status(200).json(grades);
+    })
+    .catch(error => {
+      // eslint-disable-next-line no-console
+      console.log(error);
+      res.status(500).json({ error: 'An unexpected error has occurred' });
+    });
+});
+
+app.post('/api/grades', (req, res) => {
+  const body = req.body;
+  const score = Number(req.body.score);
+  if (!body.name || !body.course || !body.score) {
+    res.status(400).json({ error: 'Entry must contain a "name", "course", and "score"' });
+    return;
+  } else if (!score || score > 100) {
+    res.status(400).json({ error: 'Invalid integer' });
+    return;
+  } else if (score <= 0) {
+    res.status(400).json({ error: 'Score must be a positive integer' });
+    return;
+  }
+  const sql = `
+    insert into "grades" ("name", "course", "score")
+    values ($1, $2, $3)
+    returning *
+  `;
+  const values = [body.name, body.course, body.score];
+  db.query(sql, values)
+    .then(result => {
+      const grades = result.rows;
+      res.status(201).json(grades);
```

Comment:
Here you want to send back a single grade object, `const grades = result.rows;` will make `grades` an array with a single object in it, instead you want to do: `const grades = result.rows[0]` so `grades` will just be an object, not in an array.